dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -5,7 +5,11 @@
       {%- do adapter.clean_up_table(relation.schema, relation.table) -%}
     {%- endif %}
     {% call statement('drop_relation', auto_begin=False) -%}
-      drop {{ relation.type }} if exists {{ relation.render_hive() }}
+      {%- if relation.type == 'view' -%}
+        drop {{ relation.type }} if exists {{ relation.render() }}
+      {%- else -%}
+        drop {{ relation.type }} if exists {{ relation.render_hive() }}
+      {% endif %}
```

Comment:
this has nothing to do with snapshot logic, could you please open another pull request for that? we aim to have minimal changes to isolate where some issues came from. Thanks.

Reply:
Removed it for him 👍 (Brecht is 🌴 )